bot/cogs/help.py

```diff
@@ -34,10 +35,13 @@ def check(r: Reaction, u: User) -> bool:
         return str(r) == DELETE_EMOJI and u.id == author.id and r.message.id == message.id
 
     await message.add_reaction(DELETE_EMOJI)
-    with suppress(HTTPException, TimeoutError):
-        _, _ = await bot.wait_for("reaction_add", check=check, timeout=300)
+
+    try:
+        await bot.wait_for("reaction_add", check=check, timeout=300)
         await message.delete()
         return
+    except (HTTPException, TimeoutError):
+        pass
 
     await message.remove_reaction(DELETE_EMOJI, bot.user)
```

Comment:
1. Looks like the `remove_reaction` belongs inside the `except` block. If moved inside, `return` and `pass` could be removed and thus make the code easier to understand.
2. Keep in mind `remove_reaction` may also raise an `HTTPException`, and that is especially more likely if attempting to delete the message raised an exception.
3. It shouldn't ignore all `HTTPException`s. It could ignore `NotFound`, the rest should get logged.